### ISSUE TITLE
HapiFHIR v5.6.0 not stable

### ISSUE THREAD

**Issue by jabahum** [iHRIS/iHRIS#376]
The version of hapi fhir v5.6.0 is quite not stable. I think you  should consider going back to a more stable one

**Comment by nobertmn** [comment]
You are right, the best hapi fair version to use right now is v5.5.1/v5.5.0

**Comment by jabahum** [comment]
@nobertmn can you indicate this in the local installation instructions, so that  when cloning  the hapi starter one can get the most stable to use for iHRIS for now
